core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -291,6 +292,7 @@ object EventLoggingListenerSuite {
   def getLoggingConf(logDir: Path, compressionCodec: Option[String] = None): SparkConf = {
     val conf = new SparkConf
     conf.set("spark.eventLog.enabled", "true")
+    conf.set("spark.eventLog.blockUpdatesEnabled", "true")
```

Comment:
Why enable this if it's not being tested?

Reply:
It is being tested. I added a test for a `SparkListenerBlockUpdated` event that won't be generated  unless the configuration is enabled.

Reply:
Ah I missed that one line.